### PR TITLE
801 add spinner supporting v2

### PR DESCRIPTION
**Pull request by norisalsaadie**
Closes https://github.com/libero/reviewer/issues/801

### CODE REVIEW

File: src/initial-submission/hooks/useSupportingFileHook.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ const hook = (
     initialValues: Submission,
     maxSupportingFiles: number,
     maxFileSize: number,
+    uploadProgressData: any,
```
Comment (will-byrne) [inline src/initial-submission/hooks/useSupportingFileHook.ts:13]: Should this be typed to something other than any?

Comment (norisalsaadie) [reply src/initial-submission/hooks/useSupportingFileHook.ts:13]: It's not typed in FileDetailsForm. So we either type it in both or leave as is

Comment (will-byrne) [reply src/initial-submission/hooks/useSupportingFileHook.ts:13]: Probably best to type it in both then.

Comment (norisalsaadie) [reply src/initial-submission/hooks/useSupportingFileHook.ts]: Types added.

**Pull request merged**